src/components/constants.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+export const FORM_ADD_PLATFORM_DIALOG = 'AddPlatformDialog'
+
+export const FORM_ADD_TOKEN_DIALOG = 'AddTokenDialog'
+
+export const FORM_ASSET_MANAGER = 'AssetManagerDialog'
+
+export const FORM_NAME = 'RevokeDialog'
+
+export const FORM_DEPOSIT_TOKENS = 'FormDepositTokens'
+
+export const FORM_RECEIVE_TOKENS = 'FormReceiveTokens'
+
+export const FORM_CBE_TOKEN = 'CBETokenDialog'
+
+export const FORM_SEND_TOKENS = 'FormSendTokens'
+
+export const FORM_CBE_ADDRESS = 'CBEAddressDialog'
+
+export const FORM_OPERATION_SETTINGS = 'OperationSettingsDialog'
+
+export const FORM_EDIT_SIGNATURES = 'FormEditSignatures'
+
+export const FORM_CREATE_EXCHANGE = 'createExchangeForm'
+
+export const FORM_EXCHANGE_BUY_TOKENS = 'ExchangeTokensForm'
+
+export const FORM_EXCHANGE_DEPOSIT_FORM = 'ExchangeDepositForm'
+export const FORM_EXCHANGE_WITHDRAWAL_FORM = 'ExchangeWithdrawalForm'
+
+export const FORM_EXCHANGE = 'ExchangeForm'
+
+export const FORM_EDIT_POLL = 'FormEditPoll'
+
+export const FORM_CUSTOM_WALLET_ADD = 'CustomWalletForm'
+
+export const FORM_MULTISIG_WALLET_ADD = 'MultisigWalletForm'
+
+export const FORM_TIME_LOCKED_WALLET_ADD = 'TimeLockedWalletForm'
+
+export const FORM_WALLET_SETTINGS = 'WalletSettingsForm'
+
+export const FORM_2FA_CONFIRM = 'Form2FAConfirm'
+
+export const FORM_2FA_ENABLE = 'Form2FAEnable'
+
+export const PROFILE_SIDE_PANEL_KEY = 'ProfileSidePanelKey'
+
+export const NOTIFICATION_SIDE_PANEL_KEY = 'NotificationSidePanelKey'
+
+export default {}
```

Comment:
It is better to remove it from my point of view. Because in this case we will loose an ability to see an error if we will try to import wrong constant from this file. 'default' may be imported as any name.